.github/workflows/release.yml

```
@@ -0,0 +1,31 @@
+name: Release
+on:
+  workflow_dispatch: {}
+  push:
+    branches:
+      - main
```

```suggestion
      - master
```

unless we are planning on updating the repo to main

---

switched to master for now